shell/browser/api/electron_api_session.cc

```diff
@@ -150,6 +154,42 @@ uint32_t GetQuotaMask(const std::vector<std::string>& quota_types) {
   return quota_mask;
 }
 
+constexpr content::BrowsingDataRemover::DataType kClearDataTypeAll =
+    0xffffffffffffffffull;
```

Comment:
- these two fields and the `ClearBrowsingDataObserver` should be [inside an unnamed namespace](https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++.md#unnamed-namespaces)

- _(nit, opinion)_ `constexpr content::BrowsingDataRemover::DataType = ~0ULL;` would be clearer

Reply:
* They are inside of the existing unnamed namespace in this file
* Agree that `~0ULL` is not only clearer but also concise 👍